README.md

```diff
@@ -1,129 +1,108 @@
 TourTrak (devcycle-mobile-app)
 ===================
 
-devcycle-mobile-app is a hybrid iOS/Android application built using Sencha Touch and Cordova for the TD Five Boro Bike Tour. To setup the dashboard/server that is used alongside this application, please refer first to our [TourTrak server repository].
+devcycle-mobile-app is a hybrid iOS/Android application built using Sencha Touch
+and Cordova for the TD Five Boro Bike Tour. To setup the dashboard/server that
+is used alongside this application, please refer first to our [TourTrak server
+repository].
 
-![screenshot](https://raw.githubusercontent.com/tofferrosen/devcycle-mobile-app/master/preview.png)
+![screenshot](https://raw.githubusercontent.com/TourTrak/devcycle-mobile-app/master/preview.png)
 
-[TourTrak server repository]: https://github.com/tofferrosen/devcycle-server.git
+[TourTrak server repository]: https://github.com/TourTrak/devcycle-server.git
 
-###Special Notes Before Setting Up Your Environment
-* <b>Cordova versions</b> can be very touchy. Install the <b>exact version</b> on this guide in order to obtain complete functionality. 
-Team Centri-Pedal Motion (RIT 2014) ran into an issue where there were a few Cordova versions that did not work with the TourTrak app. We found this by issue by browsing to the app's "About" section, clicking on "Tracking", and then clicking the "Pause/Resume" button. Upon clicking the button, there was no response by the UI. The ideal response would be a call to the backend through a cordova/phonegap plugin which would change the red "Pause" button, to a green "Resume" button. 
+## Development Instructions
+This section discusses the normal development process.  It assumes that you are
+a developer new to the application and steps through the build dependencies as
+well as the actual build process.
 
-	To see if the Cordova/Phonegap plugins are being called by the app, open `index.html` in the devcycle-mobile-app root after running `sencha app build native`. Upon hitting the Pause button, a popup dialog will show the Cordova/Phonegap call and finally the Pause button would go to a Resume button state. If it doesn't work, it is likely that Cordova is not working properly with the project.
+### Dependencies
+The following lists of dependencies *must* be installed prior to building the
+application.
 
-###Dependencies Required for Building Project
-* Git 
+* Git
 * NodeJS (Current Version)
 	* [http://nodejs.org/](http://nodejs.org/ "NodeJS Site")
 * Ruby 1.9.3
-	* (Windows) During installation, Ensure to select "Add Ruby Executables to your PATH"
+  * (Windows) During installation, Ensure to select "Add Ruby Executables to
+    your PATH"
 * Java Runtime Environment > 1.7
```

Comment:
Does 1.7 itself work? Also if we're going to say anything higher than 1.7 we should make sure this works fine on 1.8 too.